packages/terra-grid/src/Grid.module.scss

```diff
@@ -28,22 +27,57 @@
     padding-left: var(--terra-grid-gutter-padding-left, 1rem);
   }
 
-  // Create column classes
-  @for $i from 1 through 12 {
-    .column-#{$i} {
```

Comment:
Are these column classes no longer needed and written out like below?

Reply:
no, they were never used